### PR TITLE
Add details to find sku

### PR DESCRIPTION
**Pull request by r-LaForge**
This PR adds additional details on where exactly to find the SKUs - complete with screenshots of Vendor Center.

I want to link to it from the purchase CTA guide: https://vendasta.jira.com/wiki/spaces/API/pages/edit-v2/1620148234

@richard-rance @ncline-va

### CODE REVIEW

**Comment by ncline-va** [comment]
@r-LaForge the edtion_id is actually only visible to superadmins in VDC due to a decision from Levi back in the day, which I've argued against ever since that it would be 'too confusing' to partners to see the id there. 
I'm going to make an executive decision and remove that block, and make the id public. I've asked various external parties for feedback on the topic in meetings over time, and they all indicated they think it should be publicly visible.

**Comment by r-LaForge** [comment]
Thanks for fixing that Nolan! :)

File: docs/Guides/Sell/FindSKU.md [hunk 1]
```diff
@@ -5,12 +5,45 @@ We currently only support creating orders for products that don't include order
 The SKU is typically different between the demo and production environments so be sure to use the correct values.
 
 ## Your Private Products
-If your product has editions please contact support to get the SKUs for each editions. Otherwise you can get them by navigating to the product or addon in Vendor Center and looking at the URL.
 
 Product SKUs start with `MP-`. Example: `MP-c4974d390a044c28aec31e421aa662b2`
 
 Addon SKUs start with `A-`. Example: `A-GMXXNQ4ZGD`
 
+Edition SKUs start with `EDITION-`. Example `EDITION-1234`
```
Comment (richard-rance) [inline docs/Guides/Sell/FindSKU.md:13]: Edition **ids** start with `EDITION-`. Edition **SKUs** are the joined format.

```suggestion
Edition SKUs match the format `^MP-.*:EDITION-.*$`. Example `MP-c4974d390a044c28aec31e421aa662b2:EDITION-1234`
```

File: docs/Guides/Sell/FindSKU.md [hunk 2]
```diff
@@ -5,12 +5,45 @@ We currently only support creating orders for products that don't include order
...
+
+From there, the (appId) is located in the url. It starts with `MP-`:
+
+![Product SKU Url](./images/product-id-url.png)
+
+If your product does not support editions the entire editionSKU is the `MP-` prefixed id.
+
+If your app does use editions your editionSKU will have two parts:
+ - `MP-` prefixed id.
+ - `EDITION-` prefixed id.
```
Comment (richard-rance) [inline docs/Guides/Sell/FindSKU.md:32]:
```suggestion
If your app does use editions your editionSKU will have two parts joined by a colon:
 - `MP-` prefixed appId.
 - `EDITION-` prefixed editionId.
```

**Review by richard-rance — APPROVED** [review]

**Pull request merged**